src/components/overview/sub-Components/imageCarousel/carouselMain.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import ImageFullSize from './imageFullSize';
+import ArrowImage from './arrowImage';
+import ThumbnailNav from './thumbnailNav';
+
+const ImageCarousel = function(props) {
+  let arrowLeft = <div></div>;
+  let arrowRight = <div></div>;
+
```

Comment:
Interesting approach to define variables instead of conditionally rendering with ternary operators. What you're doing is probably the cleaner option when there's a lot of logic to apply.